internal/dag/dag.go

```diff
@@ -585,6 +585,18 @@ type SecureVirtualHost struct {
 	// only reason to set this to `true` is when you are migrating
 	// from internal to external authorization.
 	AuthorizationFailOpen bool
+
+	// AuthorizationBufferSettingsEnabled sets whether buffer settings are set for buffering request data sent to AuthorizationServer
+	AuthorizationBufferSettingsEnabled bool
+
+	// AuthorizationBufferSettingsMaxRequestBytes sets the maximum size of message body ExtAuthz filter will hold in-memory.
+	AuthorizationBufferSettingsMaxRequestBytes uint32
```

Comment:
Sorry, I am terrible at picking names! Naming is hard! 

@skriss please suggest! :smiley:

Reply:
Moving it to a separate struct makes naming easier and obvious.